edu_edfi_airflow/providers/snowflake/transfers/s3_to_snowflake.py

```diff
@@ -172,6 +172,13 @@ def execute(self, context):
         # Force potential string columns into lists for zipping in execute.
         if isinstance(self.resource, str):
             raise ValueError("Bulk operators require lists of resources to be passed.")
+
+        # we have to do this for the time being because the XCom that produces this list
+        # actually returns a lazily-evaluated object with no len() property
+        self.resource = list(self.resource)
+
+        if isinstance(self.table_name, str):
+            self.table_name = [self.table_name] * len(self.resource)
```

Comment:
@jayckaiser I don't think I added this code - was it removed from main recently?

Reply:
@johncmerfeld It appears that I did remove this from main in the latest [commit](https://github.com/edanalytics/edu_edfi_airflow/commit/65a2e985770bdd2b314bd6ce56273e1457b531ac#diff-64cdedd58d400346c90857524c14aa43672e219ea838d69e235a00e45574890bL175).

We need to make sure that nothing else has changed between when this was branched and current main.